.drone/pipelines/default/steps/test/cf_acceptance_tests.sh

```diff
@@ -6,41 +6,19 @@ set -o errexit -o nounset
 source ".drone/pipelines/default/runtime/binaries.sh"
 # shellcheck disable=SC1091
 source ".drone/pipelines/default/runtime/config.sh"
-
```

Comment:
I dislike the changes in this file. Ideally, we want to have this logic managed by Bazel under the `//testing/...`. This script and the smoke-tests one should be very similar and do only what they are currently doing. So the end result would be:

```
$ bazel run //testing/cf_acceptance_tests
... waiting for pod to exists
... a bunch of logs from CATS...
$ echo $?
0
```

The CI pipelines would then:

1. Deploy kubecf with CATS configured without the internetless suite.
2. `bazel run //testing/cf_acceptance_tests`
3. Delete the hanging k8s job that ran CATS.
4. Change the CATS properties to only run the internetless suite.
5. `bazel run //testing/cf_acceptance_tests`

Reply:
Note that I'm not asking to move this script and the smoke-tests to be managed by Bazel. Just try not to change it.

Reply:
So you want to inline `cats-common.sh` again ?

Reply:
So, points 1. and 2. are currently distinct steps in the Drone CI. 
My understanding right now is that you wish to have 3. and 4. also as distinct steps.
Ditto for 5., calling the same script as in step 2.
Note that your steps miss the `isolate_network` operation (and the complementary removal of same).

Personally I would move step 3 into the script for 2, and then simply remove it later, when #324 is solved upstream and in the deployment in general.

So, the full set of Drone steps you are looking for is

  1. Deploy kubecf, standard
  2. Run CATS
       - Drop hanging job
  3. Redeploy kubecf for internetless
  4. Isolate network
  5. Run CATS
  6. Remove network isolation
  7. (Redeploy kubecf for standard)

Note that in this setup failure of the 2nd CATS prevents undoing the network isolation.
The current scripts take care to undo even in presence of failure.

Reply:
Resolved with commit 3e7c18ef8275f6170fa0508edeb62fcf52201e05.
Step 4 above moved into step 3.
Left out steps 6 and 7.